### ISSUE TITLE
Common place to see latest release version

### ISSUE THREAD

**Issue by KoolPal** [leokhoa/laragon#165]
Hi,

The latest release version is just not known

https://github.com/leokhoa/laragon/releases shows version 4.0.15 which was released on Jul 4, 2019

https://sourceforge.net/projects/laragon/files/releases/ just shows 4.0 and released on September 14, 2019 and the internal version number is not known

Where is the correct location of the latest version

Thanks

**Comment by Sotiris-k** [comment]
I'd trust github than scamforge any day of the week.

**Comment by prodancer2015** [comment]
I agree with KoolPal, the current issue for Laragon is not shown properly on the official website or here at GitHub. 
See my complaint and request here: 
https://forum.laragon.org/topic/2079/current-version-of-laragon-needs-to-be-displayed-on-website

So is the developer of Laragon going to fix this soon? - After all, it is easy to adjust the main website and make a text edit here at Github to make the fix, so what is taking so long?

**Comment by leokhoa** [comment]
The official release link is on GitHub: https://github.com/leokhoa/laragon/releases
You can always find the latest download on the Laragon Download page: https://laragon.org/download
